### ISSUE TITLE
uncaught exception in login page

### ISSUE THREAD

**Issue by aledbf** [verdaccio/verdaccio#23]
```
ForbiddenError: unregistered users are not allowed to access package @myorg/demo
    at Object.allow_access (/usr/local/lib/node_modules/verdaccio/lib/auth.js:67:23)
    at next (/usr/local/lib/node_modules/verdaccio/lib/auth.js:141:7)
    at next (/usr/local/lib/node_modules/verdaccio/lib/auth.js:138:14)
    at next (/usr/local/lib/node_modules/verdaccio/lib/auth.js:138:14)
    at Auth.allow_access (/usr/local/lib/node_modules/verdaccio/lib/auth.js:146:5)
    at async.filterSeries.template.name (/usr/local/lib/node_modules/verdaccio/lib/index-web.js:52:14)
    at /usr/local/lib/node_modules/verdaccio/node_modules/async/dist/async.js:3382:13
    at replenish (/usr/local/lib/node_modules/verdaccio/node_modules/async/dist/async.js:836:21)
    at /usr/local/lib/node_modules/verdaccio/node_modules/async/dist/async.js:846:15
    at _filter (/usr/local/lib/node_modules/verdaccio/node_modules/async/dist/async.js:3381:9)
```

**Comment by mariomarin** [comment]
This bug is crashing the application when there is an authenticated user trying to access the site because you restricted the access to some package:

```
packages:
  '@*/*':
    # scoped packages
    access: $authenticated
    publish: $authenticated
```

**Comment by ben-page** [comment]
I'm having the same problem. I'm trying to upgrade from sinopia to to verdaccio. The config worked fine with sinopia. And way does a forbidden error crash the whole process? Shouldn't it just display an error or at least a 500 error.

```
storage: ./storage

listen:
    unix:/var/www/verdaccio/verdaccio.sock

url_prefix: https://npm.private.com

web:
    title: Private NPM

auth:
  htpasswd:
    file: ./htpasswd
    max_users: -1

# a list of other known repositories we can talk to
uplinks:
  npmjs:
    url: https://registry.npmjs.org/

packages:
  '@private/*':
    # scoped packages
    access: $authenticated
    publish: $authenticated

  '*':
    access: $all
    publish: $authenicated
    proxy: npmjs

# log settings
logs:
  - {type: stdout, format: pretty, level: http}
  #- {type: file, path: verdaccio.log, level: info}
```

**Comment by trentearl** [comment]
The fix should be in 2.1.0. I havent been able to recreate the error.

Feel free to open this back up if you have a problem
